client/src/app/[locale]/layout.tsx

```diff
@@ -0,0 +1,36 @@
+import "./globals.css";
+
+import { getTranslations } from "next-intl/server";
+
+export async function generateMetadata({ params: { locale } }: { params: { locale: string } }) {
+  const t = await getTranslations({ locale });
+
+  return {
+    title: t("Rangelands Data Platform"),
+    description: t(
+      "Diverse ecosystems crucial for both wildlife and people. Explore their beauty and significance with our Vital Ecosystem Atlas, advocating for their protection and restoration.",
+    ),
+    alternates: {
+      canonical: "/",
+      languages: {
+        en: "/en",
+        es: "/es",
+        fr: "/fr",
+      },
+    },
+  };
+}
+
+export default async function LocaleLayout({
+  children,
+  params: { locale },
+}: {
+  children: React.ReactNode;
+  params: { locale: string };
+}) {
+  return (
+    <html lang={locale}>
+      <body>{children}</body>
```

Comment:
By adding the provider to the main layout, all your components have access to the messages:
```suggestion
      <body>
        <NextIntlClientProvider messages={messages}>
          {children}
        </NextIntlClientProvider>
      </body>
```